config/default.yml

```
@@ -98,6 +98,21 @@ Rails/ActionFilter:
     - app/controllers/**/*.rb
     - app/mailers/**/*.rb
 
+Rails/ActionName:
+  Description: Use only specific action names.
+  Enabled: pending
```

I understand the motivation, but it's probably too strict for some user applications and should be disabled by default.

```suggestion
  Enabled: false
```

---

Yeah, I know I'm the one who enabled this, but I agree to disable this cop by default 👌 
I certainly believe this is a good practice, but it is also true that most existing Rails apps do not follow this practice for now.

---

I totally agree that it's good practice in the Rails controller design. But sometimes the real world is hard 😅